pkg/render/render.go

```diff
@@ -34,6 +34,27 @@ func RenderFile(renderPath, templatePath string, cfg interface{}) error {
 	}
 	defer renderFile.Close()
 
+	// NOTE For some reason, the template file for keepalived.conf has the
```

Comment:
IIRC, this was because of a bug in MCO. They were hard-coding permissions that included executable for no apparent reason.